docs/overview-of-optee-rust-examples.md

```diff
@@ -0,0 +1,31 @@
+---
+permalink: /trustzone-sdk-docs/overview-of-optee-rust-examples
+---
+
+# Overview of OP-TEE Rust Examples
+
+All OP-TEE Rust examples are suffixed with `-rs`, which work as standalone host application and corresponding TA (Trusted Application) and can be found in separate directories.
```

Comment:
Normally, we break lines in 80 characters (even for markdownn document). This helps tools (like git diff) to show changes between different versions.

You can simply use `gq` in `vim` to break lines. [1]

[1] https://vim.fandom.com/wiki/Format_only_long_lines